### PR TITLE
Add option for recording notification and thus foreground service.

### LINKED ISSUE #33: Call startForeground() in TelecineService.onStartCommand()
On my Nexus 5, I've noticed Telecine being randomly killed (during recording) by the system. Calling startForeground() should go a long way to preventing this, I think.

### PR DESCRIPTION
**Pull request by JakeWharton**
Closes #33.

### CODE REVIEW

File: telecine/src/main/java/com/jakewharton/telecine/TelecineService.java [hunk 1]
```diff
@@ -9,9 +10,12 @@
 import javax.inject.Provider;
 import timber.log.Timber;
 
+import static android.app.Notification.PRIORITY_MIN;
+
 public final class TelecineService extends Service {
   private static final String EXTRA_RESULT_CODE = "result-code";
   private static final String EXTRA_DATA = "data";
+  private static final int NOTIFICATION_ID = 99118822;
```
Comment (mattprecious) [inline telecine/src/main/java/com/jakewharton/telecine/TelecineService.java]: What's with the random ids?

**Comment by mattprecious** [comment]
LGTM

File: telecine/src/main/java/com/jakewharton/telecine/TelecineService.java (hunk 1 above)
Comment (JakeWharton) [inline telecine/src/main/java/com/jakewharton/telecine/TelecineService.java]: Is there a better way? I've always just picked random values.

Comment (mattprecious) [inline telecine/src/main/java/com/jakewharton/telecine/TelecineService.java]: I just start at 0 and increment. Doesn't really matter, was just curious.

**Pull request merged**